maquetado/scss/_titles.scss

```diff
@@ -0,0 +1,10 @@
+.title {
+  background: transparent url('assets/triangle.svg') no-repeat center left;
+  color: $titles-color;
+  font-family: $main-font;
+  font-size: 24px;
+  height: 93px;
```

Comment:
Assign this value in the `_variables.scss` file

Reply:
Hola Henry... ¿Este cambio se refiere a colocar el `height` en una variable?